core/src/validator.rs

```diff
@@ -844,6 +844,16 @@ impl Validator {
             "Using: block-verification-method: {}, block-production-method: {}",
             config.block_verification_method, config.block_production_method
         );
+        if matches!(
+            config.block_production_method,
+            BlockProductionMethod::ThreadLocalMultiIterator
+        ) {
+            warn!(
+                "--block-production-method thread-local-multi-iterator is deprecated \
+                   and will be removed in a future release. Please use \
```

Comment:
If we know it will be v2.2, we could mention that.

I'm also wondering if we should retire this entire option (specifying of scheduler), or do you think we'll need more options again in the future?

Reply:
Ryo has plans to adapt unified scheduler at some point